lib/matplotlib/figure.py

```diff
@@ -1044,70 +1044,97 @@ def fixlist(args):
         key = fixlist(args), fixitems(kwargs.items())
         return key
 
+    @docstring.dedent_interpd
     def add_axes(self, *args, **kwargs):
         """
         Add an axes to the figure.
 
-        Call signature::
+        Call signatures::
```

Comment:
You can actually remove those call signatures. We don't use those anymore, as this information is contained in the numpydoc arguments list.

Reply:
I believe that those are sometimes still necessary when there are actually several more or less independent ways to call a function.

Reply:
Well… it's definitely a sign that the API for those particular functions are way to complicated, but there isn't much we can do about that these days.

Reply:
I agree. It would also be good to get rid of many of the (*args, **kwargs).